app/tests/mocks/window.js

```diff
@@ -19,7 +19,8 @@ function (_, Backbone, NullStorage) {
       href: window.location.href,
       search: window.location.search,
       pathname: '/',
-      origin: window.location.origin
+      origin: window.location.origin,
+      host: window.location.host
```

Comment:
I could swear `host` is not available universally, and `hostname` is, can you ensure this works in IE10?